tools/swift-machine-learning-api/codes/trainer/__init__.py

```diff
@@ -0,0 +1 @@
+"""Empty Initialization file"""
```

Comment:
unnecessary docstring remove to make this an empty __init__.py .

Reply:
Modified.